templates.go

```diff
@@ -60,9 +60,9 @@ type From struct {
 // Options specifies settings to apply to this Template.
```

Comment:
According the Go's [doc guide](https://blog.golang.org/godoc-documenting-go-code) Paragraph 5&6 the comment should start with `TemplOptions` instead of `Options`